### PR TITLE
EDASH-40 

### PR DESCRIPTION
**Pull request by alexbaeza**
## Description
Added Birthday widget
Added Cool Loading Spinner
Fixing indentations

## JIRA Ticket (if applicable)
https://transformuk.atlassian.net/browse/EDASH-40

## Screenshots
Screenshot 2019-09-20 at 10 32 09

### CODE REVIEW

File: src/containers/Birthdays/Birthdays.scss [hunk 1]
```diff
@@ -0,0 +1,17 @@
+.birthday-wrapper {
+  height: 100%;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  min-height: 16em;
+  overflow: auto;
+}
+.no-cake {
```
Comment (martin-bucinskas) [inline src/containers/Birthdays/Birthdays.scss]: WHAT DO YOU MEAN NO CAKE???? 😱

Comment (alexbaeza) [reply src/containers/Birthdays/Birthdays.scss]: Removed as it's no longer in use

File: server/gsheets.js [hunk 2]
```diff
@@ -233,6 +238,44 @@ const retrieveAllReminders = async auth => {
...
+      },
+      (err, res) => {
+        if (err) return console.log(`The API returned an error: ${err}`);
+
+        const rows = res.data.values;
+
+        if (rows.length) {
+          rows.map(row => {
+            //filter to make sure that only people with consent to "Other communication"
+            if (row[4].toUpperCase() === 'YES') {
```
Comment (martin-bucinskas) [inline server/gsheets.js]: Think using filters would convey this better? reduces few lines and is quite readable IMO
```js
rows.filter(row => row[4].toUpperCase() === 'YES')
    .map(row => birthdays.push({ name: row[0], date: row[1] }));
```

**Review by martin-bucinskas — COMMENTED** [review]
LGTM, maybe change the filtering to actual filter method?

File: server/gsheets.js (hunk 2 above)
Comment (alexbaeza) [reply server/gsheets.js]: Totally agree

Comment (alexbaeza) [reply server/gsheets.js]: Thanks

**Review by martin-bucinskas — APPROVED** [review]

**Pull request merged**